docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,190 @@
+==============================
+Tutorial: Moving a Data Volume
+==============================
+
+.. note:: This tutorial takes roughly 30 minutes, but because there are a few things to download, times might vary depending on the speed of your connection.
+
+.. contents:: 
+   :local:
+   :backlinks: none
+   :depth: 2
+
+You will use Flocker to migrate a Docker container with its data volume from one host to another.
+The container you move will be part of a two-container application, the other container will not move and the two will remain connected even when they are on different hosts.
+
+To begin the tutorial you will first install the Flocker client on your local machine, then install Flocker onto two hosts.
+You will then be ready to use Flocker to migrate a Docker container with a volume attached from one host to the other.
+
+You will be controlling your Flocker cluster via the CLI you've installed locally.
+The following diagram illustrates the initial server-side Flocker setup that you will control via the CLI:
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :width: 60 %
+   :alt: In the initial server-side Flocker setup there are two servers, one of which has two Docker containers running; one container is a running a web application, the other has a Redis database with a volume.
+   :align: center
+
+The following diagram illustrates how the server-side Flocker setup will be configured at the end of the tutorial:
+
+.. image:: images/flocker-tutorial-final-setup.svg
+   :width: 60 %
+   :alt: Following the completion of this tutorial the server-side Flocker setup will be configured with the web application still running within a container on the first server, while the Redis server with a volume is running on the second server.
+   :align: center
+
+Flocker manages the data migration and the link between the two containers.
+
+If you have any feedback or problems, you can :ref:`talk-to-us`.
+
+Before You Begin
+================
+
+Requirements
+------------
+
+To replicate the steps demonstrated in this tutorial, you will need:
+
+* Ubuntu 14.04 or OS X with `Homebrew`_ installed.
```

Comment:
We now have instructions for Ubuntu 15.04. I think that it is worth looking at the log for the GSG again to see what has changed.